content/news/core-monthly/2023/core-monthly-2023-07-01-2023-07-31.md

```diff
@@ -0,0 +1,530 @@
+---
+layout: post
+title:  "PrestaShop Core Monthly - July 2023"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2023-08-09
+authors: [ kpodemski ]
+image: /assets/images/theme/banner-core-monthly.jpg
+twitter_image: /assets/images/theme/banner-core-monthly.jpg
+icon: icon-calendar
+tags:
+ - core-monthly
+---
+
+Read about what has been going on in PrestaShop Project in July 2023. It was a good month with over 340 merged pull requests! :tada:
+
+![Core Monthly banner](/assets/images/theme/banner-core-monthly-wide.jpg)
+
+Even in summer we see a lot of great progress in PrestaShop Project. In July we merged over 340 pull requests.
+
+We welcomed three new committers:
+- Guyomar Alexis ([@ga-devfront](https://github.com/ga-devfront))
+- Clément Désiles ([@jokesterfr](https://github.com/jokesterfr))
+- Morgan Pichat ([@M0rgan01](https://github.com/M0rgan01))
+
+You can see all current project members on the updated [People and Roles page](https://www.prestashop-project.org/project-organization/people-and-roles/) on the project's website.
+
+Now, let's see what happened in July from the technical point of view!
+
+### PrestaShop 8.1.1. The first patch version and upgrade issues.
+
+In July we worked to prepare the first patch version for PrestaShop 8.1.x branch. It contains [a lot of bug fixes and small improvements](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+milestone%3A8.1.1+). Most notable changes are related to the [checkout process validation](https://github.com/PrestaShop/PrestaShop/pull/33156), there are also some [fixes regarding upgrades](https://github.com/PrestaShop/PrestaShop/pull/33164) from previous versions of PrestaShop.
+
+We are discovering more issues related to the upgrade process from previous versions to PrestaShop 8.1. One repeatedly reported issue is related to the SSL configuration on Windows. You can read more about this issue [here](https://github.com/PrestaShop/PrestaShop/issues/33286). The one proposed way to fix this issue:
+
+1. Open php.ini for currently enabled PHP version.
+2. Find your SSL cert file, you can see some common names here: https://github.com/composer/ca-bundle/blob/main/src/CaBundle.php#L85
+3. Set path to your cert: `openssl.cafile = "path_to_the_crt_file";`
+
+We also noticed that on some servers upgrade fails if you perform it on PHP 8.1. We will investigate this issue and try to fix it in the next patch version. For the time being, if you encounter some problems, it's recommended to try on PHP 7.4 instead of PHP 8.1. You can change the PHP version after the upgrade.
```

Comment:
```suggestion
We also noticed that, on certain servers, the upgrade fails if you perform it on PHP 8.1. We will investigate this issue and try to fix it in the next patch version. For the time being, if you encounter some problems, it's recommended to try on PHP 7.4 instead of PHP 8.1. You can change the PHP version after the upgrade.
```